Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -317,6 +319,8 @@ void HTMLInputElement::did_pick_color(Optional<Color> picked_color)
         m_value = value_sanitization_algorithm(picked_color.value().to_deprecated_string_without_alpha());
         m_dirty_value = true;
 
+        MUST(m_color_well_element->style_for_bindings()->set_property(CSS::PropertyID::BackgroundColor, m_value));
```

Comment:
(I think this case is safe because you shouldn't be able to get a `did_pick_color()` callback without the element being attached to the document. See other comments if this doesn't make sense.)

Reply:
You actually do since you can make the element without attaching it to the rest of the DOM in JavaScript, which means it doesn't have this associated element

I did not check if Pixedit still works 😅